services/SlackService.test.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-new */
```

Comment:
Would it work if this eslint rule was disabled on line 17 instead of for the whole file?

Reply:
I'll move it there